Firestore/core/src/model/mutable_document.h

```diff
@@ -212,7 +212,10 @@ class MutableDocument {
   DocumentKey key_;
   DocumentType document_type_ = DocumentType::kInvalid;
   SnapshotVersion version_;
-  std::shared_ptr<const ObjectValue> value_ = std::make_shared<ObjectValue>();
+  // Using a shared pointer to ObjectValue makes MutableDocument copy-assignable
+  // without having to manually create a deep clone of its Protobuf contents
```

Comment:
Ultranit: please add a full stop at the end of the sentence.

Reply:
Done